### ISSUE TITLE
whitelist does not work in test environment

### ISSUE THREAD

**Issue by xyingsoft** [Betterment/webvalve#30]
At first, thanks for this great gem and efforts.
I need to enable some urls in the test environment - for instance, webdrivers gem is blocked by webvalve, so all test is fail now.

Looking through the codebase I found following code.
`def webmock_disable_options
      { allow_localhost: true }.tap do |opts|
        opts[:allow] = whitelisted_url_regexps unless Rails.env.test?
      end
    end`

And I couldn't enable any urls in test environment now since it coded that :allow option work only in test environment.

I think it would be great if :allow option work for all environments.

**Comment by samandmoore** [comment]
@xyingsoft i'm closing this issue as a duplicate of #29 please follow along there.
